_gsdocs-proposals/2020/proposal_ROOTgeneric.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Generic Proposal Title
+layout: gsoc_proposal
+project: ROOT
```

Comment:
ROOT already confirmed?

Reply:
No, this is just as a template, clearly stated now in the "For HSF projects and mentors" section. There is also a guideline for sending proposals now there